### PR TITLE
Pg11 support

### PR DESCRIPTION
**Pull request by jburel**
Add support for pg11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Role Variables
 
 Defaults: `defaults/main.yml`
 
-- `postgresql_version`: The PostgreSQL major version: `9.4` (default), `9.5`, `9.6` or `10`
+- `postgresql_version`: The PostgreSQL major version: `9.4` (default), `9.5`, `9.6`, `10`, `11`
```
Comment (joshmoore) [inline README.md]: I assume changing the default would be a breaking change.

Comment (jburel) [reply README.md]: I initially thought about that and reverted the change since it will be breaking

Comment (manics) [reply README.md]: To reduce the testing matrix I think this addition of PG11 is a good time to drop 9.4 (and 9.5 too?), especially as the matrix will soon be increased further when Ubuntu 18.04 is added.

Comment (joshmoore) [reply README.md]: And remove the default all together and force listing a version?

Comment (manics) [reply README.md]: I think making the version mandatory is a good idea.

**Comment by joshmoore** [comment]
https://github.com/ome/omero-server-docker/pull/42/commits/e7d20df3f5d6ee0342dbed6b89b9d70ed5d7a36f passed. Propose releasing as 4.0.0

**Pull request merged**